routes/index.js

```diff
@@ -150,7 +150,8 @@ router.get('/search', function(req, res, next) {
     var query = new RegExp('.*' + searchString + '.*', 'im');
 
     function findSongs(username) {
-      return function (done) {
+        consloe.log(username);
```

Comment:
drop this line.

Reply:
I will.
I forgot to delete after testing on console.